server/controllers/shoutoutController.js

```diff
@@ -0,0 +1,16 @@
+const environment = process.env.NODE_ENV || 'development';
+const config = require('../knexfile')[environment];
+const db = require('knex')(config);
```

Comment:
I think that these should probably stay in App and then you can just import `db` into the controller